### ISSUE TITLE
Contribute command to trigger "Fetch from remote(s)"

### ISSUE THREAD

**Issue by nqminhuit** [mhutchie/vscode-git-graph#412]
Hi there,

Could you please tell me the command to trigger "Fetch from remote(s)" here:

![image](https://user-images.githubusercontent.com/7464067/98897272-a0423d00-24dd-11eb-90c0-5e3796992f60.png)

Many thanks for this awesome extension!
Best regards,

**Comment by mhutchie** [comment]
Hi @nqminhuit,

"Fetch from remote(s)" currently only has a button, there's no keyboard shortcut to trigger it (like "Open a Terminal for this Repository" & "Repository Settings").

The reason why there currently isn't a keyboard shortcut for "Fetch from remote(s)" is that:
* No other users have previously requested it.
* The only keyboard shortcut I could think of that users would naturally try to use is `CTRL + F` (i.e. Fetch), however this is already used for the Find Widget.

I'd be happy to add a keyboard shortcut for "Fetch from remote(s)" if you can propose one that:
* Isn't already used in the Git Graph View (`CTRL+F` - Find, `CTRL+H` - Scroll to Head, `CTRL+R` - Refresh, or `CTRL+S` - Scroll to Stash)
* Isn't already used by a frequently used Visual Studio Code Keyboard Shortcut.
* It's likely that many users would naturally try to use it (without having to read about it first).

**Comment by nqminhuit** [comment]
Hi @mhutchie,

Thank you for quick reply!

My question was not about asking for the keyboard shortcut because it might conflict with other commands in vscode, it was for the command that trigger it.

For example: something like: `"command": "git-graph.fetchFromRemotes"` then we will map it ourself:
```
{
    "key": "ctrl+k ctrl+r",
    "command": "git-graph.fetchFromRemotes"
  },
```

The reason I'm asking this command is that there is a nice dialog indicates that vscode is fetching from remotes and close when it finishes, this dialog provides me the visual indication that my code is being fetched. Built-in vscode git command fetch does not show me anything so I don't know if it fetches complete or not.

**Comment by mhutchie** [comment]
I see, thank you for clarifying your request.

Currently Git Graph doesn't contribute any commands to Visual Studio Code that result in a Git action being triggered via the Git Graph View. However, it would definitely be possible to implement one for `git-graph.fetchFromRemotes` in an upcoming release.

Would the follow behaviour meet your requirements?
1\. User triggers the command `git-graph.fetchFromRemotes` from the Visual Studio Code Command Palette, or a custom mapped keyboard shortcut.
2a\. If the workspace only contains a single repository, open the Git Graph View for that repository.
2b\. If the workspace contains multiple repositories, first display a Visual Studio Code Quick Pick allowing the user to select a repository, before opening the repository in the Git Graph View. _(the Quick Pick would default to the latest repository opened in the Git Graph View)_
3\. Once loaded, trigger "Fetch from remote(s)" (and display the existing Git Graph View progress dialog).

**Comment by nqminhuit** [comment]
To me, the behaviors are great!

**Comment by mhutchie** [comment]
Thanks for confirming @nqminhuit,

I'll include this in an upcoming release, I'm sure many users will value it!

**Comment by mhutchie** [comment]
This will be available in v1.28.0.

If you'd like to use it before the next release, you can download [v1.28.0-beta.2](https://github.com/mhutchie/vscode-git-graph/releases/tag/v1.28.0-beta.2), and install it following the instructions provided [here](https://github.com/mhutchie/vscode-git-graph/wiki/Downloading-&-Installing-Releases#manual-installation).
